apps/desktop-wallet/src/components/AppHeader.tsx

```diff
@@ -185,6 +185,7 @@ const HeaderButtons = styled.div`
   align-items: center;
   gap: var(--spacing-1);
   -webkit-app-region: no-drag;
+  padding-right: ${platform.isWindows ? '165px' : 0};
```

Comment:
Maybe linux needs this too?

Reply:
This needs to be changed to

```suggestion
  padding-right: ${platform.isMac ? '165px' : 0};
```